packages/sit-onyx/src/components/OnyxCheckboxGroup/types.ts

```diff
@@ -0,0 +1,46 @@
+import type { OnyxCheckboxProps } from "../OnyxCheckbox/types";
+
+export type OnyxCheckboxGroupProps<
```

Comment:
Do you mean for the options property or the whole checkbox group?

Reply:
We should align both so that we have a common baseline interface for all selection controls. Idea is that you can easily swap out a combobox, with a radiogroup and with a checkboxgroup.
Therfore they must have the same property subset.
I approved your PR, we can follow up on this in #334.